packit/cli/init.py

```diff
@@ -45,12 +46,49 @@
 @click.option(
     "-f", "--force", is_flag=True, help="Reset config to default if already exists."
 )
+@click.option(
+    "--upstream-url",
+    help="URL or local path to the upstream project URL; "
+    "this implies creating a source-git repo",
+)
+@click.option(
+    "--upstream-ref",
+    help="Use this upstream git ref as a base for your source-git repo",
+)
+@click.option("--fedora-package", help="Pick spec file from this Fedora Linux package")
+@click.option(
+    "--centos-package",
+    help="Pick spec file from this CentOS Linux or CentOS Stream package",
+)
+@click.option(
+    "--dist-git-branch",
+    help="Get spec file from this downstream branch, "
+    "for Fedora this defaults to master, for CentOS it's c8s",
```

Comment:
Document the default when `--dist-git-path` is used.

Reply:
good point, the arg is actually ignored when path is defined